src/util/favicon.js

```diff
@@ -0,0 +1,88 @@
+import { DOMParser } from "@xmldom/xmldom";
```

Comment:
This file has to be updated given the changes made on the original favicon PR https://github.com/MetaMask/metamask-mobile/pull/6951

Reply:
➿ @tannn-younet @cuonglt-ync @viphan007